alluxio-ui/README.md

```diff
@@ -85,3 +85,6 @@ NOTE: `coverage-ci` is meant to run tests once and quit (for continuous integrat
 
     This will also generate a coverage report within each package: `common/coverage/lcov-report/index.html`, `master/coverage/lcov-report/index.html`, `worker/coverage/lcov-report/index.html`. You may also run `coverage-ci` instead of `coverage` in this step if you would like this to execute only once.
 
+### Shrinkwrapping dependencies
+
+Once changes are tested and things work as planned, please run `npm shrinkwrap && lerna exec npm shrinkwrap` to lock package dependency versions.
```

Comment:
Yes, good point, I chatted with Calvin on this. I think this will be part of the release process. I will add to this summary.

Reply:
@ZacBlanco I've added a general description, PTAL.

Reply:
This is definitely better but just reading the above section doesn't exactly tell me what "shrinkwrapping dependecies" means. Maybe this is a personal preference but I just don't understand what the term "shrinkwrapping" means in this context.

Reply:
I'm open to suggestions, but the short story is that it's related to that cli command.

Reply:
So if I'm understanding correctly then after this part

> Once a dependency changes, it is important to verify any functionality that could be affected by the change. This can be done via unit testing, system testing, and/or old fashioned QA testing.

I would write:

"In order to keep the UI consistent for production builds we shrinkwrap our dependecies using `npm-shrinkwrap` so that our package versions stay consistent across builds. Shrinkwrapping locks dependency versions so that updates to underlying UI libraries don't affect our build."

> Once changes are tested and things work as planned, please run `npm shrinkwrap && lerna exec npm shrinkwrap` to lock package dependency versions.

Reply:
I like that, so I'll grab it.